logging/monolog_exclude_http_codes.rst

```diff
@@ -0,0 +1,61 @@
+.. index::
+   single: Logging
+   single: Logging; Exclude HTTP Codes
+   single: Monolog; Exclude HTTP Codes
+
+How to Configure Monolog to Exclude Specific HTTP Codes from the Log
+====================================================================
+
+Sometimes your logs become flooded with unwanted HTTP errors, for example,
+403s and 404s. When using a ``fingers_crossed`` handler, you can exclude
+logging these HTTP codes based on the MonologBundle configuration:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+        monolog:
+            handlers:
+                main:
+                    # ...
+                    type: fingers_crossed
+                    handler: ...
+                    excluded_http_codes: [403, 404, { 400: ['^/foo', '^/bar'] }]
+
```

Comment:
I think this should use the multi line format - one status code per line - I think it will be more clear.

Reply:
I'm not sure about this suggestion because I like how compact this looks ... but I don't have a strong opinion about this.

Reply:
I don't mind either way. I think multi-line might be a little clearer.

Reply:
What is the preferred format for multi-line arrays in YAML?

A:
```yaml
excluded_http_codes: [
    403, 
    404, 
    { 400: ['^/foo', '^/bar'] }
]
```

or B:
```yaml
excluded_http_codes: 
    - 403
    - 404
    - 
        400: 
            - ^/foo
            - ^/bar
```

Reply:
@weaverryan don't you think the expanded format is less readable in this case? The empty space in the last element in the B) notation is confusing ... and the A) format is a bit confusing too because of the way the brackets are displayed.